File: Modul 6/soal_1/PRAK601-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>

int main(){
    int row = 0, col = 0;
    scanf("%d %d", &row, &col);

    int matrix[row][col];
    for(int i = 0; i < row; ++i){
        for(int j = 0; j < col; ++j){
            scanf("%d", &matrix[i][j]);
        }
    }

    for(int i = 0; i < row; ++i){
        for(int j = 0; j < col; ++j){
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul 6/soal_1/PRAK601-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
row, col = map(int, input().split())

input_matriks = list(map(int, input().split()))

pointer = 0
for i in range(0, row) :
    for j in range(0, col):
        print(input_matriks[pointer], end=" ")
        pointer += 1
    print()
